etl/RetrosheetParser.py

```diff
@@ -60,25 +56,16 @@ def parse_id(self, retro_list):
                 }
         return id_instance
         
-#         # Why not this? Is it more an aesthetic than a practical thing?
-#         id_instance = {
-#                 'game_id' : retro_list[1]
-#                 , 'team': retro_list[1][:2]
-#                 , 'year' : retro_list[1][3:6]
-#                 , 'month' : retro_list[1][7:8]
-#                 , 'day' : retro_list[1][9:10]
-#                 , 'game_num' : retro_list[1][11]
-#                 }
     
     def parse_version(self, retro_list):
         "Parse the version type"
         # Indicated on retrosheet.org that the version record is obsolete.
         pass
 
+
     def parse_info(self, retro_list):
     	"Parse the info type"
-    	# We don't need to read all "info" rows. For example, an "info" row repeats the game date, which is already collected from "id".  There are other random "info" types that are not particularly relevant. Perhaps it is better to trim those less relevant info lines later?
```

Comment:
As I mentioned above, I think it makes sense to keep the transformation of the raw retrosheet data as light as possible.  This parsing relies entirely on the position of elements in a poorly structured csv file, which will be brittle and prone to breaking. Pulling everything in, when it's not much data and no more work, makes good sense and it allows us to separate the parsing logic from the downstream logic about the data format that makes most sense for analysis.